pkg/sloop/ingress/kubeclient.go

```diff
@@ -12,25 +12,27 @@ import (
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/clientcmd/api"
 )
 
 // GetKubernetesContext takes optional user preferences and returns the Kubernetes context in use
-func GetKubernetesContext(masterURL string, kubeContextPreference string) (string, error) {
+func GetKubernetesContext(masterURL string, kubeContextPreference string, PrivilegedAccess bool) (string, error) {
```

Comment:
please do not capitalize it, it makes it global variable

Reply:
Hi @sana-jawad The review is at 35 commits, could you please review changes from all commits. This was taken up earlier .